package.json

```diff
@@ -38,7 +38,7 @@
     "ts-jest": "^25.0.0",
     "tsd": "0.11.0",
     "typescript": "^3.7.5",
-    "vue": "^3.0.0-rc.11",
+    "vue": "^3.0.0-rc.12",
```

Comment:
nit: I think it would be better to also update the compiler-sfc version above

Reply:
oops, yes.

Reply:
done!